src/resources/develop/Currencies.yaml

```diff
@@ -1765,15 +1779,43 @@ values:
         - NER
         - SEN
         - TGO
-      monetary_authority: Central Bank of West African States
+      monetary_authority: Central Bank of the West African States
+
+  - code: XPD
+    name: PalladiumTroyOunce
+    label: Palladium troy ounce
+    metadata:
+      symbol: "Pd"
+      precision: 4
+
+  - code: XPF
+    name: CFPFranc
+    label: CFP franc
+    metadata:
+      symbol: "FCFP"
+      precision: 0
+      fractions: 100
+      fractional_unit: centime
+      polity:
+        - PYF
+        - NCL
+        - WLF
+      monetary_authority: Overseas Issuing Institute
+
+  - code: XPT
+    name: PlatinumTroyOunce
+    label: Platinum troy ounce
+    metadata:
+      symbol: "Pt"
+      precision: 4
```

Comment:
Do we want anybody to be able to create an “ Platinum troy ounce credit” instrument?

Reply:
Resolution: should be added to this list for completeness. Can add separate filtering for CRN selection.